app/src/main/java/com/cronus/zdone/WorkTimeManager.kt

```diff
@@ -1,27 +1,23 @@
 package com.cronus.zdone
 
-import android.content.Context
-import android.content.SharedPreferences
+import com.cronus.zdone.api.TasksRepository
+import io.reactivex.Observable
 import javax.inject.Inject
 import javax.inject.Singleton
 
 @Singleton
-class WorkTimeManager @Inject constructor(context: Context) {
```

Comment:
no need for context, as we are no longer storing work time locally (context gave us access to on-device key-value store)